products/tarantool.md

```diff
@@ -0,0 +1,68 @@
+---
+permalink: /tarantool
+layout: post
+title: Tarantool
+command: $ tarantool --version
+releasePolicyLink: https://www.tarantool.io/en/doc/latest/release/policy/
+changelogTemplate: |
+  https://www.tarantool.io/en/doc/latest/release/__LATEST__/
+category: db
+eolColumn: Support Status
+releaseDateColumn: false
+sortReleasesBy: 'releaseCycle'
+releases:
+  - releaseCycle: "2.8"
+    release: 2021-12-22
+    eol: false
+    latest: "2.8.3"
+
+  - releaseCycle: "2.7"
+    release: 2021-08-19
+    eol: 2021-08-19
+    latest: "2.7.3"
+
+  - releaseCycle: "2.6"
+    release: 2021-04-21
+    eol: 2021-04-21
+    latest: "2.6.3"
+
+  - releaseCycle: "2.5"
+    release: 2020-12-30
+    eol: 2020-12-30
+    latest: "2.5.3"
+
+  - releaseCycle: "2.4"
+    release: 2020-10-22
+    eol: 2020-10-22
+    latest: "2.4.3"
+
+  - releaseCycle: "2.3"
+    release: 2020-07-17
+    eol: 2020-07-17
+    latest: "2.3.3"
+
+  - releaseCycle: "2.2"
+    release: 2019-12-31
+    eol: 2020-04-20
+    latest: "2.2.2"
+
+  - releaseCycle: "1.10"
+    release: 2021-12-22
+    eol: false
+    latest: "1.10.12"
```

Comment:
Going by the text below, this is an LTS release. Please mark the current LTS as well.

```suggestion
    latest: "1.10.12"
    lts: true
```